docs/basics/app-anatomy.md

```diff
@@ -158,7 +158,7 @@ Each module should also implement the `RegisterServices` method as part of the [
 
 #### Handlers
 
-The [handler](../building-modules/handler.md) refers to the part of the module responsible for processing the `Msg` after it is routed by `baseapp`. Handler functions of modules are only executed if the transaction is relayed from Tendermint by the `DeliverTx` ABCI message. If the transaction is relayed by `CheckTx`, only stateless checks and fee-related stateful checks are performed. To better understand the difference between `DeliverTx`and `CheckTx`, as well as the difference between stateful and stateless checks, click [here](./tx-lifecycle.md).
+The [handler](../building-modules/msg-services.md#handler-type) refers to the part of the module responsible for processing the `Msg` after it is routed by `baseapp`. Handler functions of modules are only executed if the transaction is relayed from Tendermint by the `DeliverTx` ABCI message. If the transaction is relayed by `CheckTx`, only stateless checks and fee-related stateful checks are performed. To better understand the difference between `DeliverTx`and `CheckTx`, as well as the difference between stateful and stateless checks, click [here](./tx-lifecycle.md).
```

Comment:
Do we still want to use the word handler?

Reply:
We had agreed to remove any mention to handler in the docs, except the "building modules" section, which @amaurymartiny did as part of this PR https://github.com/cosmos/cosmos-sdk/pull/7782 which isn't merged yet.